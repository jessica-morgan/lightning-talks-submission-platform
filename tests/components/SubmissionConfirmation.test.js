import React from 'react'
import SubmissionConfirmation from '../../client/components/SubmissionComfirmation'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'
import { Provider } from 'react-redux'
import configureStore from 'redux-mock-store'
import thunk from 'redux-thunk'

test('SubmissionConfirmation renders correctly', () => {
  const mockStore = configureStore([thunk])({
    auth: [{ loggedIn: true, error: false }],
    submissions: [{ submissionError: false }]
  })

  const component = renderer.create(
    <Provider store={mockStore}>
      <MemoryRouter>
        <SubmissionConfirmation />
      </MemoryRouter>
    </Provider>
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
