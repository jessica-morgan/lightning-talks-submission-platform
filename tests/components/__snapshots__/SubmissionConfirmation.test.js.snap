// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmissionConfirmation renders correctly 1`] = `null`;
